src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
```

Comment:
@amb26 - can you expand a little on how the default can be overriden? I assume we'd want something similar to a configuration variable like this pseudocode:

```
if(PREFERNATIVEWIDGETS) {
fluid.makeGradeLinkage("fluid.textfieldSliderNativeWidget", ["fluid.textfieldSlider"],["fluid.textfieldSliderNative"]);
} else {
fluid.makeGradeLinkage("fluid.textfieldSliderNativeWidget", ["fluid.textfieldSlider"],["fluid.textfieldSliderJQuery"]);
}
```

Then we follow that pattern for other components where we need the preference for native / jQuery to be used?